### PR TITLE
fix(DB/Creature): InhabitType of ICC flying Creatures

### PR DESCRIPTION
**Pull request by bronzato**
Ids: 37955,38434,38435,38436,37501,38197,37502,38198,37232,38362
Was with wrong InhabitType making then fly on the floor

Closes AzerothCore issue 4661
https://github.com/azerothcore/azerothcore-wotlk/issues/4661

## Changes Proposed:
-    Changing InhabitType for some flying creatures inside ICC (Some Nerubians and Blood Queen Boss)

## Issues Addressed:
- Closes issue 4661
- https://github.com/azerothcore/azerothcore-wotlk/issues/4661

## Tests Performed:
- No Erros
- Tested in game with the current build of AzerothCore
- In the Blood Queen Boss she body was half in the floor with the Inhabittype 4, i change to 1 and she in on the floor in right place, and she still can fly in the air phase (the creature script do the job)
- Tested in Ubuntu Server 20.04

## How to Test the Changes:

1.Go to ICC
2.Go to Blood Queen Boss
3.She body is not half on the floor
4.Nerubians before Sindragosa Boss, should be on the floor to (after descending from the roof in the web)

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Yehonal — APPROVED** [review]
Good one! approved!

Can someone test it please? (all the bosses inside the **IN** list)

**Comment by FrancescoBorzi** [comment]
> Good one! approved!
> 
> Can someone test it please? (all the bosses inside the **IN** list)

These are the NPCs to check:

```
37955,38434,38435,38436,37501,38197,37502,38198,37232,38362
```

cc @azerothcore/testers

**Comment by morpheux** [comment]
@FrancescoBorzi 

Works for me.

**Review by Helias — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
good job @bronzato and thanks @morpheux for testing this!

File: data/sql/updates/pending_db_world/rev_1625085335146756200.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1625085335146756200');
+
+UPDATE creature_template SET InhabitType = 1 WHERE entry IN (37955,38434,38435,38436,37501,38197,37502,38198,37232,38362);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1625085335146756200.sql:3]: backticks
https://www.azerothcore.org/wiki/sql-standards

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1625085335146756200.sql (hunk 1 above)
Comment (FrancescoBorzi) [reply data/sql/updates/pending_db_world/rev_1625085335146756200.sql]: I added them myself

**Pull request merged**